### PR TITLE
Add support for BIG SOLIDUS unicode characters in off topic names

### PR DESCRIPTION
**Pull request by minalike**
This PR will allow us to add these unicode characters to off-topic names: 

- ⧸ Big Solidus

- ⧹ Big Reverse Solidus

image

The PR for the site should be merged first: python-discord/site#717

### CODE REVIEW

**Review by Robin5605 — APPROVED** [review]
thank you for this incredibly important PR!

**Review by shenanigansd — APPROVED** [review]

File: bot/converters.py [hunk 1]
```diff
@@ -382,8 +382,8 @@ async def convert(self, ctx: Context, duration: str) -> datetime:
 class OffTopicName(Converter):
     """A converter that ensures an added off-topic name is valid."""
 
-    ALLOWED_CHARACTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ!?'`-<>"
-    TRANSLATED_CHARACTERS = "𝖠𝖡𝖢𝖣𝖤𝖥𝖦𝖧𝖨𝖩𝖪𝖫𝖬𝖭𝖮𝖯𝖰𝖱𝖲𝖳𝖴𝖵𝖶𝖷𝖸𝖹ǃ？’’-＜＞"
+    ALLOWED_CHARACTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ!?'`-<>⧹⧸"
```
Comment (kosayoda) [inline bot/converters.py:385]: `ALLOWED CHARACTERS` should have the regular forward/backward slashes, no? Unless I'm going mad, what you have now are the SOLIDUSses

**Review by kosayoda — CHANGES_REQUESTED** [review]

File: bot/converters.py (hunk 1 above)
Comment (minalike) [reply bot/converters.py:385]: Oh, right. I made the functionality actually use the unicode characters and tested using them as well. Be back in a minute

Comment (minalike) [reply bot/converters.py]: Tested this time with actual regular slashes and pushed changes 
image

**Review by kosayoda — APPROVED** [review]
rock solidus

**Pull request merged**